### PR TITLE
Add metadata store and basic unikernel service

### PR DESCRIPTION
**Pull request by Crazykev**
1. add container/sandbox metadata store
2. add basic service skeleton

Will bring container and VM workflow when test is enough
@feiskyer @resouer PTAL

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,9 +15,9 @@ before_install:
 
 install:
   - docker run -v /usr/local/bin:/target jpetazzo/nsenter
-  - go install -v $(go list ./... | grep -v /vendor/)
   - hack/install-protoc.sh
   - hack/install-hypercontainer.sh
+  - go install -v $(go list ./... | grep -v /vendor/)
```
Comment (feiskyer) [inline .travis.yml]: why this change?

**Comment by feiskyer** [comment]
Also this PR needs a rebase now.

File: .travis.yml (hunk 1 above)
Comment (Crazykev) [reply .travis.yml]: @feiskyer I've introduce libvirt-go dependency in this patch, when go install `libvirt-go`, it will try to build it, and it need libvirt-dev lib to do that. We installed this lib in `hack/install-hypercontainer.sh`, so I changed this order to install libvirt-dev first. 

BTW, new patch is on the way, thanks for reviewing.

Comment (feiskyer) [reply .travis.yml]: OK, that makes sense. thanks
